frontend/webpack.config.js

```diff
@@ -73,4 +75,8 @@ const config = {
 	},
 };
 
+if (process.env.BUNDLE_ANALYSER === 'true') {
```

Comment:
const plugin = // all required Plugins 

if(devEnv){
plugin.push(bundleAnalyser)
}

then use plugins

Reply:
devEnv refers to the NODE_ENV variable? As we have separate webpack config files for prod and dev. So how could this condition be used?

Reply:
sorry my bad here i was referring `devEnv` to  

```
process.env.BUNDLE_ANALYSER === 'true'
```

Reply:
Okay alright!